spec/features/sign_up_spec.rb

```diff
@@ -5,4 +5,10 @@
     visit('/')
     expect(page).to have_button 'Sign up'
   end
+
+  scenario 'Allow user to enter details' do
```

Comment:
These two feature test probably could have been merged into one. In this test, capybara is clicking the signup button in order to let the user enter details. If the page didn't have the sign up button (which the first feature test is testing for), then the test would still fail, so you're still covering yourself.